.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
We shouldn't commit the `.idea` folder, please remove it. For bonus points, please change `.gitignore` to simply ignore the whole `.idea` folder (like [this](https://github.com/ConduitIO/conduit/blob/7f64ff1e351f40ab6b3074b0253cd511992d989c/.gitignore#L17)).